manila/network/neutron/neutron_network_plugin.py

```diff
@@ -301,6 +305,20 @@ def deallocate_network(self, context, share_server_id):
         for port in ports:
             self._delete_port(context, port)
```

Comment:
we don't need to call `_delete_port` here, since we are going to delete them anyway below?

Reply:
replied below